File: src/main/java/de/tomjuri/examplemod/ExampleCommand.java
package de.tomjuri.examplemod;

import cc.polyfrost.oneconfig.utils.commands.annotations.Command;
import cc.polyfrost.oneconfig.utils.commands.annotations.Main;
import net.minecraft.client.Minecraft;
import net.minecraft.util.ChatComponentText;

@Command(value = "examplecommand")
public class ExampleCommand {

    @Main
    private void main() {
        Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText("§c" + ExampleModConfig.exampleText));
    }
}

File: src/main/kotlin/de/tomjuri/examplemod/ExampleCommand.kt
package de.tomjuri.examplemod

import cc.polyfrost.oneconfig.utils.commands.annotations.Command
import cc.polyfrost.oneconfig.utils.commands.annotations.Main
import de.tomjuri.examplemod.mixin.MinecraftInvoker
import net.minecraft.client.Minecraft
import net.minecraft.util.ChatComponentText

@Command(value = "examplecommand")
class ExampleCommand {

    @Main
    private fun main() {
        Minecraft.getMinecraft().thePlayer.addChatMessage(ChatComponentText("§c${ExampleMod.config.exampleText}"))
    }
}